docs/installation.rst

```diff
@@ -48,6 +48,17 @@ If muFFT_ is unable to find the NetCDF libraries (the `FileIONetCDF` class
 is missing), then set the environment variables `NETCDFDIR` (for serial
 compile) or `PNETCDFDIR` (for parallel compiles, to e.g. `$USER/.local`).
 
+Sometimes the installation fails because muFFT_ attempts to compile with `MPI` support but not all necessary libraries are
+avaible. If you do not need `MPI` support, you can manually disable it in the following way:
+
+```
+python3 -m pip install muFFT --install-options="--disable-option"
```

Comment:
Should this be "--disable-mpi"?

Reply:
of course yes thanks for pointing out...